tests/test_firewalld.py

```
@@ -60,305 +54,293 @@ def firewalld_disabled(mock_settings_disabled):
-    
+
     def test_cidr_validation_ipv4_missing_mask(self):
         """Test CIDR validation fails for IPv4 without mask."""
         settings = {
             "firewalld": {
                 "enabled": True,
-                "monitored_ips": ["192.168.1.1"]  # Missing /32
+                "monitored_ips": ["192.168.1.1"],  # Missing /32
             }
         }
         with pytest.raises(ValueError, match="IPv4 address.*must include network mask"):
```

Ruff (0.15.13)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)